scripts/release.js

```diff
@@ -50,6 +50,7 @@ const run = async () => {
             .replace('[[commit]]', commit)
             .replace('[[release_url]]', `<a href="${releaseUrl}">${releaseUrl}</a>`)
             .replace('[[notes]]', releaseNotes)
+            .replace(/<\/?p>/ig, '\n')
```

Comment:
This wouldn't match if the `<p>` had any attributes, assuming you've tested this and that doesn't occur anyway?

Reply:
Yeah, I can't see that happening. Thanks.